docker/cygnus-ngsi/cygnus-entrypoint.sh

```diff
@@ -582,7 +582,18 @@ if [ "$CYGNUS_ORION_HOST" != "" ]; then
 fi
 
 # Check if POSTGIS ENV vars
-if [ "$CYGNUS_POSTGIS_HOST" != "" ]; then
+if [ "${CYGNUS_POSTGIS_ENABLE_VOLUME_CONFIG,,}" == "true" ]; then
+    AGENT_CONF_FILE=agent_postgis.conf
```

Comment:
What kind of config are in your agent_postigs.conf that are not in conf env vars? Which other conf files a re in your volume?

Reply:
Good question, an important note, the volume only mount files, so, we can mount the `agent_postgis.conf`, `name_mappings_postgis.conf` and `grouping_rules_postgis.conf`.

Reply:
This is an example extracted from the `docker-compose.yml`:

```yaml
iot-cygnus:
  container_name: iot-cygnus
  hostname: iot-cygnus
  image: telefonicaiot/fiware-cygnus:latest
  volumes:
    - ${DBPATH_POSTGIS_AGENT}:/opt/apache-flume/conf/agent_postgis.conf
    - ${DBPATH_POSTGIS_MAPPING}:/opt/apache-flume/conf/name_mappings_postgis.conf
environment:
    - CYGNUS_POSTGIS_ENABLE_VOLUME_CONFIG =true
...
```

Reply:
I'd suggest to use a name for the env var closer to what it actually does. For instance:

```
CYGNUS_POSTGIS_SKIP_CONF_GENERATION
```

From my point of view, the possibility of putting the configuration in a volume is *a consequence* of skipping configuration generation.

In addition, this env var should be documented at https://github.com/telefonicaid/fiware-cygnus/blob/master/doc/cygnus-ngsi/installation_and_administration_guide/install_with_docker.md, in the Postgis section.

Reply:
Yep, definitely, your proposal is more accurate, the change is [here](https://github.com/telefonicaid/fiware-cygnus/pull/1657/commits/ab6eadab326b5cb1844d3cc263fd6f1b09a4fde0).